src/Infrastructure/Service/Initializer.php

```diff
@@ -12,15 +12,16 @@
 use SprykerSdk\Sdk\Core\Application\Dependency\Repository\TaskYamlRepositoryInterface;
 use SprykerSdk\Sdk\Core\Application\Dependency\TaskManagerInterface;
 use SprykerSdk\Sdk\Core\Application\Dto\ReceiverValue;
+use SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver;
 use SprykerSdk\SdkContracts\Entity\SettingInterface;
 use SprykerSdk\SdkContracts\Entity\SettingInterface as EntitySettingInterface;
 
 class Initializer implements InitializerInterface
 {
     /**
-     * @var \SprykerSdk\Sdk\Infrastructure\Service\CliValueReceiver
+     * @var \SprykerSdk\Sdk\Infrastructure\Service\ValueReceiver\CliReceiver
      */
-    protected CliValueReceiver $cliValueReceiver;
+    protected CliReceiver $cliValueReceiver;
```

Comment:
It must be depended on the abstraction. Please change it to the ReceiverInterface interface.